paper.tex

```diff
@@ -136,6 +136,34 @@ \subsection{Refunding}
 
 Where $price$  is the subscription price, $M$ is the number of expected data samples, $F_{b}$ is the brokerage fee(\%), $F_{t}$ is the transaction fee of the smart contract(\%), $N$ is the number of subscribers in this contract.
 
+\subsection{Game Theory Evaluation}
+
+\subsubsection{Game Theory Scenario}
+In our model there are two kinds of participants. One is data provider, and the other one is data buyer. Data provider would receive subscription fee for buying data each time the data is transmitted to data buyer.
+The data buyer will keep buying data from data povider, only when the data sold by the data provider with high enough correct rate, $\pi_{correct}$. We deonte this correct rate threshold with $\pi_{threshold}$.
+Once the correct rate is lower than $\pi_{threshold}$, the data buyers would consider this data provider as a malicious data provider, and reject to buy the data from this data provider.
+
+\subsubsection{Model Description}
+We use a repeated game to express the scenario of our Data Marketplace. Figure.\ref{fig:decision_tree} shows the decision tree to depict the the repeated game we used. Each level in this decision tree represents each round of data transmission from data provider to data buyers, and the data buyers would pay $p_{fee}$ in total to buy the data.
```

Comment:
Do we have to explain how "repeated game" works for Game Theory?

Reply:
?

Reply:
So I should present some simple instroction of repeated game here ?

Reply:
For formal writing, you shall explain in  advance.